src/test/test-execute.c

```diff
@@ -551,8 +559,14 @@ static void test_exec_capabilityambientset(Manager *m) {
                 return;
         }
 
-        test(m, "exec-capabilityambientset.service", 0, CLD_EXITED);
-        test(m, "exec-capabilityambientset-merge.service", 0, CLD_EXITED);
+        if (have_effective_cap(CAP_NET_ADMIN) <= 0 ||
+            have_effective_cap(CAP_NET_RAW) <= 0) {
+                log_error("Skipping %s, this process does not have enough capabilities", __func__);
```

Comment:
I'm not sure about `log_error`. At other places in `test-execute`, `log_notice` is used to indicate events of this type.

Reply:
OK.